app/javascript/hyacinth_ui_v1/components/digital_objects/rights/ItemRightsForm.js

```diff
@@ -155,7 +171,7 @@ class ItemRightsForm extends React.Component {
       },
     } = this.state;
 
-    const { id } = this.state;
+    const { data: { uid } } = this.props;
```

Comment:
I know I go back and forth between using `uid` and `id` in the frontend. But I think we should really move towards using `id.` It's a pattern i've also been trying to follow with the graphql endpoints. No need to change it here since this will have to be converted to a functional component in the near future.

Reply:
Isn't it easier to go with `uid` because it's the actual name of the property on the digital object, so it'll play better with the common destructuring assignment use case?

Reply:
The way I imagine it, the graphql endpoint would not return a `uid`, it would just return an `id`. While we are hitting the RESTful endpoint `uid` makes sense, but once we move to the graphql endpoint we should start using `id`.

Reply:
Maybe we should change:
`metadata_attribute :uid, Hyacinth::DigitalObject::TypeDef::String.new`
to:
`metadata_attribute :id, Hyacinth::DigitalObject::TypeDef::String.new` ?

Reply:
I would 100% approve of that. Is it possible to do without colliding with DigitalObjectRecord?

Reply:
I think it should be fine as long as we don't delegate DigitalObject::Base's :id to DigitalObjectRecord.  Will be good to do in the near future.